__tests__/unit/media-play-button.spec.js

```diff
@@ -0,0 +1,145 @@
+import { spy } from 'sinon';
```

Comment:
Why all the underscores in `__tests__`?

Reply:
`__tests__` is a default for Jest, pretty common: https://github.com/facebook/jest/issues/637